openstack/identity/v3/users/requests.go

```diff
@@ -61,3 +73,67 @@ func Get(client *gophercloud.ServiceClient, id string) (r GetResult) {
 	_, r.Err = client.Get(getURL(client, id), &r.Body, nil)
 	return
 }
+
+// CreateOptsBuilder allows extensions to add additional parameters to
+// the Create request.
+type CreateOptsBuilder interface {
+	ToUserCreateMap() (map[string]interface{}, error)
+}
+
+// CreateOpts implements CreateOptsBuilder
+type CreateOpts struct {
+	// Name is the name of the new user.
+	Name string `json:"name" required:"true"`
+
+	// DefaultProjectID is the ID of the default project of the user.
+	DefaultProjectID string `json:"default_project_id,omitempty"`
+
+	// Description is a description of the user.
+	Description string `json:"description,omitempty"`
+
+	// DomainID is the ID of the domain the user belongs to.
+	DomainID string `json:"domain_id,omitempty"`
+
+	// Enabled sets the user status to enabled or disabled.
+	Enabled *bool `json:"enabled,omitempty"`
+
+	// Extra are free-form extra key/value pairs to describe the user.
+	Extra map[string]interface{} `json:"-"`
+
+	// Options are coded 4-character options defined by the API.
```

Comment:
You're too quick :)

Yes, I'll fix this. In the Keystone code, these are actually stored as 4-character codes. But then I found out that the readable-name is used and forgot to change this.